codespell_lib/data/dictionary.txt

```diff
@@ -11937,6 +11945,7 @@ higeine->hygiene
 higer->higher
 higest->highest
 high-affort->high-effort
+highe->high, higher,
 highes->highest
```

Comment:
I know it's not your change, but could you also add `highs` to this as a suggestion.